blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -201,6 +229,27 @@ public void handle(ChannelHandlerContext ctx, FullHttpRequest request) {
         }
     }
 
+    private String prepareErrorResponse(String tenantId, String message) {
```

Comment:
If its okay with you, I will handle this in future PR's where I have to refactor this anyway.